test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -72,7 +72,7 @@ MATCHER_P(NewBlockMatcher,
 TEST_F(SynchronizerTest, ValidWhenInitialized) {
   // synchronizer constructor => on_commit subscription called
   EXPECT_CALL(*consensus_gate, on_commit())
-      .WillOnce(Return(rxcpp::observable<>::empty<Block>()));
+      .WillOnce(Return(rxcpp::observable<>::empty<std::shared_ptr<shared_model::interface::Block>>()));
```

Comment:
please, clang-format if haven't done it yet

Reply:
it's formatted